test/unit/org/apache/cassandra/service/RetryStrategyTest.java

```diff
@@ -64,6 +69,26 @@ public void fuzzParser()
         });
     }
 
+    @Test
+    public void testSeededWaitRandomizer()
+    {
+        long wait1 = generateSeededWait(100);
+        long wait2 = generateSeededWait(100);
+        Assertions.assertThat(wait1).isEqualTo(wait2);
+
+        long wait3 = generateSeededWait(200);
+        Assertions.assertThat(wait1).isNotEqualTo(wait3);
```

Comment:
This test is not validating the change that you made in the patch. how about changing it to something like this
```suggestion
        RetrySpec spec = new RetrySpec(new RetrySpec.MaxAttempt(10),
                                       new DurationSpec.LongMillisecondsBound("200ms"),
                                       new DurationSpec.LongMillisecondsBound("1000ms"));

        long wait1 = RetrySpec.toStrategy(createContext(100), spec).computeWait(1, TimeUnit.MILLISECONDS);
        long wait2 = RetrySpec.toStrategy(createContext(100), spec).computeWait(1, TimeUnit.MILLISECONDS);
        long wait3 = RetrySpec.toStrategy(createContext(200), spec).computeWait(1, TimeUnit.MILLISECONDS);

        Assertions.assertThat(wait1).isEqualTo(wait2);
        Assertions.assertThat(wait1).isNotEqualTo(wait3);
```

Reply:
Thank you, fixed